### PR TITLE
Typo fix

### PR DESCRIPTION
**Pull request by tobie**

### CODE REVIEW

File: IP_POLICY_GUIDANCE.md [hunk 1]
```diff
@@ -67,8 +67,8 @@ and a [corporate CLA][CCLA].
 Projects can choose to adopt either the individual CLA by itself or
 both the individual CLA and the corporate CLA without any further review.
 
-If you need to use a different CLA, you will need to
-[obtaining an exemption from the board](#obtaining-an-exemption-from-the-board).
+If you need to use a different CLA, you must
+[obtain an exemption from the board](#obtaining-an-exemption-from-the-board).
```
Comment (ljharb) [inline IP_POLICY_GUIDANCE.md:71]: nit:
```suggestion
If you need to use a different CLA, you must [obtain an exemption from the board](#obtaining-an-exemption-from-the-board).
```
hard wrapping makes for much noisier diffs

**Review by ljharb — APPROVED** [review]

**Review by Trott — APPROVED** [review]

File: IP_POLICY_GUIDANCE.md (hunk 1 above)
Comment (tobie) [reply IP_POLICY_GUIDANCE.md:71]: This doc uses [semantic line breaks](https://sembr.org/). Whether that's a good idea or not is debatable, but let's stay consistent in this PR. Happy to +1 a separate PR that changes the whole doc if people prefer a different style.

Comment (ljharb) [reply IP_POLICY_GUIDANCE.md:71]: That says "A semantic line break may occur before and after a hyperlink." - note "may" and not "must" :-) but sure, a nit is a nonblocker

Comment (tobie) [reply IP_POLICY_GUIDANCE.md:71]: Point taken!

Comment (tobie) [reply IP_POLICY_GUIDANCE.md:71]: (Also I can’t believe you actually read this.)

**Review by joesepi — APPROVED** [review]

**Review by eemeli — APPROVED** [review]

**Pull request merged**